ls1intum/Artemis
# Changed Max Score into Points, set bonus to 0 default

### Description
Allow the instructors to set bonus points for exercises #2053
This is a change for instructors being able to add bonus points.
I changed max score to points and set bonus default to 0. 

### Steps for Testing
1. check that bonus is always set to zero.
2. Check on exercise testing that Max Score is now call points. 



### Test Coverage
